config/locales/en.yml

```diff
@@ -36,6 +36,8 @@ en:
         create: "Add"
   activerecord:
     attributes:
+      gsa18f/procurement:
+        recurring: "Recurring charge"
```

Comment:
I vote that we either try to move as much to the locales or try not to use them at all. Being split on I18n could get kinda confusing :hammer: